src/main/java/org/eclipse/pass/schema/SchemaServlet.java

```diff
@@ -15,27 +41,42 @@
 @WebServlet("/schemaservice")
 public class SchemaServlet extends HttpServlet {
     private static final long serialVersionUID = 1L;
+    private List<String> repository_list;
```

Comment:
The repository_list and next fields here should not be fields in the class. It looks like they should be local variables to a method. Remember there could be multiple requests coming into the servlet simultaneously.